Conectado a MongoDB: rescue
Índices de usuarios creados correctamente
Índices de usuarios creados correctamente
Índices de usuarios creados correctamente
 * Serving Flask app 'app'
 * Debug mode: on
Address already in use
Port 5002 is in use by another program. Either identify and stop that program, or start the server with a different port.
